src/main/proto/uri.proto

```diff
@@ -20,20 +20,94 @@
  */
 syntax = "proto3";
 
-package uprotocol;
+package uprotocol.v1;
 
 import "google/rpc/status.proto";
 import "google/protobuf/any.proto";
 
-option java_package = "org.eclipse.uprotocol";
+option java_package = "org.eclipse.uprotocol.v1";
+option java_outer_classname = "UUriProto";
 option java_multiple_files = true;
 
 
+// Data representation of uProtocol <b>URI</b>.
+// This class will be used to represent the source and sink (destination) parts 
+// of the Packet, for example in a CloudEvent Packet. <br>
+// UUri is used as a method to uniquely identify devices, services, and resources
+// on the  network.<br>
+// Where software is deployed, what the service is called along with a version and
+// the resources in the service. Defining a common URI for the system allows 
+// applications and/or services to publish and discover each other.<br>
+message UUri {
```

Comment:
do we want to explicitly have the type of UUri?
not sure, just thinking about it. If someone has it set explicitly, it might make it easier to understand what is inside.
uPlatform needs to use this Uri for routing

Reply:
The datamodel won't have this (at least for now) it will be in the wrapper function classes added to the SDKs